tutorials/GMAW_process.ipynb

```diff
@@ -145,20 +146,15 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "buffer = weldx.asdf.util._write_buffer(tree)\n",
-    "weldx.asdf.util.notebook_fileprinter(buffer)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
+   "metadata": {
```

Comment:
pycharm sucks for notebooks :)

Reply:
Can't we add a proper treatment of that to the `clean_notebooks` script? Maybe we can also handle this python version ping pong too.